desktop/core/src/desktop/api_public.py

```diff
@@ -159,6 +161,16 @@ def get_history(request):
   django_request = get_django_request(request)
   return notebook_api.get_history(django_request)
 
+@api_view(["POST"])
+def analyze_table(request, dialect, database, table, columns=None):
+  django_request = get_django_request(request)
+
+  if dialect in ["impala", "beeswax"]:
+    return beeswax_api.analyze_table(django_request, database, table, columns=None)
+  
```

Comment:
```
else:
  return HttpResponse()
```

Reply:
Where to put this in docs? Maybe here https://docs.gethue.com/developer/api/rest/#table-details-and-columns ? @romainr